### PR TITLE
Update requests version in requirements.txt

### PR DESCRIPTION
**Pull request by Dk-A-r**
Update requests version in requirements.txt

### CODE REVIEW

File: requirements.txt
Comment (OrlovAlexandr) [inline requirements.txt:1]: correct version

**Review by OrlovAlexandr — COMMENTED** [review]
everything is fine

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ torchaudio==0.13.0
 speechbrain==0.5.13
 soundfile==0.11.0
 pytest==5.4.1
-requests
```
Comment (OrlovAlexandr) [inline requirements.txt:7]: approved

**Review by OrlovAlexandr — APPROVED** [review]
approved

**Review by dzailz — APPROVED** [review]
Looks fine! Merge ASAP

**Pull request merged**